### ISSUE TITLE
Camera black screen on Android 9 webview

### ISSUE THREAD

**Issue by bkhurram** [MABelanger/react-html5-camera-photo#29]
the camera show the black screen with Android Pie into webview.

on previous version of android with webview it's work well.

i had search and then i found the solution. if you add attribute muted="muted" to the tag <video>  its work good.
```
<video
          style={videoStyles}
          ref={this.videoRef}
		  autoPlay={true}
		  muted="muted"
          playsInline
        />
```

**Comment by alexanderkjeldaas** [comment]
This is also true on desktop Chrome 73.

**Comment by MABelanger** [comment]
This fix is merged on npm version 1.4.1
